Plugins~/Src/MeshSyncClientBlender/python/unity_mesh_sync_preferences.py

```diff
@@ -45,17 +126,32 @@ def editors_path_exists(self):
 
     def is_hub_installed():
         path = msb_get_hub_path()
-        return dir is not None and os.path.exists(path)
+        return os.path.exists(path)
+
+    def is_hub_supported():
+        hub_version = msb_get_hub_version()
+        if hub_version == "":
+            return False
+
+        return msb_get_most_recent_version(hub_version, "3.0.0") == hub_version
```

Comment:
> Does this check if the current hub version is 3.0.0?

No, msb_get_most_recent_version(a, b) returns the most recent version between a and b. So, if hub_version > 3.0.0 it will return the hub_version. If hub version < 3.0.0 it will return 3.0.0.  
 

> What happens if we're at a higher patch level?

If its 3.0.0 or a minor or major above it, it will return True. If its anything bellow 3.0.0 it will return False.